### ISSUE TITLE
Some normalize() properties are not camelCased

### ISSUE THREAD

**Issue by blicksky** [styled-components/polished#332]
- `polished` version: 1.9.3
- `glamor` version: 2.20.40

Mixin/Helper/Shorthand you were using and how you were using it:

```javascript
const normalizeStyles = normalize();

Object.entries(normalizeStyles).forEach(([selector, rules]) =>
  css.global(selector, rules)
);
```

What you are seeing:

Warnings are printed to the console due to non-camelCase properties being used:

> Warning: Unsupported style property background-color. Did you mean backgroundColor?
Warning: Unsupported style property -webkit-text-decoration-skip. Did you mean WebkitTextDecorationSkip?
Warning: Unsupported style property -webkit-appearance. Did you mean WebkitAppearance?

What you expected to see:

I expected to see no such warnings.

Reproduction:

https://codesandbox.io/s/4wlvyyv32x?expanddevtools=1

This demonstrates that while styles still seem to be applied correctly (the background color of the link is transparent as opposed to red, even though the `normalize()` style is defined with `'background-color'`), a warning is still printed to the console.

https://github.com/styled-components/polished/pull/170 seems to have attempted to address this issue, but some properties were overlooked.

**Comment by bhough** [comment]
@blicksky Thanks for catching this. Since `normalize` is getting a decent sized update in 2.0, I fixed it as part of that release with #334.

**Comment by blicksky** [comment]
Thanks! I noticed that you used a lowercase 'w' in "webkitAppearance" in your changes. I believe that the vendor prefixed properties need to start with an uppercase letter. A quick test produced this warning:

> Warning: Unsupported vendor-prefixed style property webkitAppearance. Did you mean WebkitAppearance?
